docs/calendar.md

```diff
@@ -36,6 +36,46 @@ const calendar = new Intl.DateTimeFormat().resolvedOptions().calendar;
 date.withCalendar(calendar).add({ months: 1 });
 ```
 
+### Invariants Across Calendars
+
+The following "invariants" (statements that are always true) hold for all built-in calendars, and SHOULD hold for properly-authored custom calendars:
+
+- `year`, `month`, `day` are always integers that increase as time goes forward
+- `month` and `day` are always positive integers
```

Comment:
This might need to be phrased a bit differently for `month` and `day`, how about this?
```suggestion
- `year` is always an integer that increases as time goes forward
- `month` and `day` are always positive integers that increase as time goes forward, except they reset at the boundary of a year or month, respectively
```